### PR TITLE
refactor createWordGroup function

### PR DESCRIPTION
**Pull request by hankeliu2015**
refactor createWordGroup function with functional programming method

### CODE REVIEW

File: src/components/new-landing-page/hero.tsx [hunk 1]
```diff
@@ -11,15 +11,26 @@ const phrases: ReadonlyArray<string> = [
...
-const wordGroups: AnimateWords = [
-  { color: 'human-blue', words: ['more', 'join', 'Boost', 'Meet', 'Learn', 'Use', 'Mentor', 'Live'] },
-  { color: 'human-pink', words: ['human', 'international', 'Your', 'Amazing', 'New', 'Latest', 'Diverse', 'Your'] },
-  { color: 'human-blue', words: ['internet', 'community', 'Resume', 'People', 'Skills', 'Technology', 'Community', 'Values'] },
-]
+const createWordGroups = function(phrasesArray: ReadonlyArray<string>): readonly object[] {
+  return phrasesArray.reduce((AnimateWordsArray, phrase, index) => {
+    const wordsArray = phrase.split(' ')
+    if (wordsArray.length !== 3) {
+      throw new Error (`The ${index}th phrase (${phrase}) need to be exact 3 words`)
```
Comment (will-weiss) [inline src/components/new-landing-page/hero.tsx]: Let's fix this typo, it should say "exactly"

**Review by will-weiss — CHANGES_REQUESTED** [review]
Small change requested, looks great!

File: src/components/new-landing-page/hero.tsx (hunk 1 above)
Comment (hankeliu2015) [reply src/components/new-landing-page/hero.tsx]: > Let's fix this typo, it should say "exactly"

typos corrected.

Comment (hankeliu2015) [reply src/components/new-landing-page/hero.tsx]: Got this TypeScript warning, I think it is related to the way I define wordGroups. 
Screen Shot 2022-03-17 at 5 41 52 PM

**Review by will-weiss — APPROVED** [review]

**Pull request merged**